libusb/libusbi.h

```diff
@@ -310,6 +310,8 @@ int usbi_vsnprintf(char *dst, size_t size, const char *format, va_list args);
 #define LIBUSB_PRINTF_WIN32
 #endif /* defined(_MSC_VER) && (_MSC_VER < 1900) */
 
+#define PRINTF_FORMAT(a,b) __attribute__((__format__ (__printf__, a, b)))
```

Comment:
shouldn't it be under `ifder`/compiler-specific?
I believe this is `__GNUC__`-specific - MSVC doesn't support it

Reply:
and I believe we need an `#else` fallback as well